File: platform/src/org/siani/goros/box/ui/displays/templates/NodePage.java
package org.siani.goros.box.ui.displays.templates;

import io.intino.alexandria.core.Box;
import org.siani.goros.box.ui.displays.notifiers.NodePageNotifier;

public class NodePage<DN extends NodePageNotifier, B extends Box> extends AbstractNodePage<DN, B> {

    public NodePage(B box) {
        super(box);
    }

}

File: platform-ui/src/displays/templates/NodePage.js
import React from "react";
import { withStyles } from '@material-ui/core/styles';
import AbstractNodePage from "../../../gen/displays/templates/AbstractNodePage";
import NodePageNotifier from "../../../gen/displays/notifiers/NodePageNotifier";
import NodePageRequester from "../../../gen/displays/requesters/NodePageRequester";
import DisplayFactory from 'alexandria-ui-elements/src/displays/DisplayFactory';
import { withSnackbar } from 'notistack';

const styles = theme => ({});

export default class NodePage extends AbstractNodePage {

	constructor(props) {
		super(props);
		this.notifier = new NodePageNotifier(this);
		this.requester = new NodePageRequester(this);
	};


}

DisplayFactory.register("NodePage", withStyles(styles, { withTheme: true })(withSnackbar(NodePage)));